src/MuTalk-Model/MTPragmaTestFilter.class.st

```diff
@@ -36,6 +36,22 @@ MTPragmaTestFilter >> filterTests: aTestCaseCollection [
 	^ self subclassResponsibility
 ]
 
+{ #category : 'accessing' }
+MTPragmaTestFilter >> filteredTestReason [
+
+	| reasonString |
+	reasonString := self reasonString.
+
+	^ pragmaArguments isEmpty
+		  ifTrue: [ reasonString , '>' ]
+		  ifFalse: [
+			  | pragmaString |
+			  pragmaString := pragmaArguments reduce: [ :arg1 :arg2 |
+				                  arg1 , ' ' , arg2 ].
+
+			  reasonString , pragmaString , '>' ]
+]
```

Comment:
This method is weird... why is it attaching `>` at the end of the string?

Reply:
Because `reasonString` is the start of the string, and it contains `<`.
It's because I wanted the pragma to be between <>, like this `<pragma>`